apps/client/src/router/index.ts

```diff
@@ -189,13 +202,27 @@ router.beforeEach(async (to, _from, next) => {
   const validPath = ['Login', 'Home', 'Doc', 'NotFound', 'ServicesHealth']
   const snackbarStore = useSnackbarStore()
   const userStore = useUserStore()
+  const systemStore = useSystemStore()
   userStore.setIsLoggedIn()
 
+  await systemStore.listSystemSettings('maintenance')
+
+  // Redirige vers la page maintenance si la maintenance est activée
+  if (
+    typeof to.name === 'string'
```

Comment:
Y'a une raison pour tester le type ici ?

Reply:
to.name peut être un string ou un symbole